src/main/webapp/resources/js/components/Buttons/AddMemberButton.jsx

```diff
@@ -19,9 +19,9 @@ const { Text } = Typography;
 export function AddMemberButton({
   label,
```

Comment:
I only added a classname to this file to fix some flaky tests to get this very PR to pass. I've started converting it, but I require some help on this one. Please see 13ff1fc6e6dbb4a8ed61b89e2a38722b3780991a.

Reply:
Totally my mistake then, sorry.  I actually did not notice that it was just a class name you added.  I do appreciate you updating it for me now since it is a small file, thank you.